xarray/core/dataset.py

```diff
@@ -1114,8 +1130,9 @@ def sel_points(self, dim='points', method=None, tolerance=None,
         Dataset.isel_points
         DataArray.sel_points
         """
-        pos_indexers = indexing.remap_label_indexers(
-            self, indexers, method=method, tolerance=tolerance)
+        pos_indexers, new_indexes = indexing.remap_label_indexers(
```

Comment:
if we ignore `new_indexes` here (which is fine, certainly for now), use the variable name `_` instead to indicate that it's unused.